### ISSUE TITLE
Overlap doesn't use cellcolor for entire cell

### ISSUE THREAD

**Issue by jamespharvey20** [andmarti1424/sc-im#343]
New user.  So glad you added the `overlap` option a while ago.

For a cell that overlaps, the overlapping content's colors are based on the cells they overlap on, rather than the actual cell with the content.

In other words, if you have all of column A colored red, and the rest of the sheet white, if A5 overlaps to the right, it's only red for the portion visually within column A.  The portions that are visually in a column to the right would be white in this example, even though all of the text is within A.

Every GUI spreadsheet I've ever used keeps the formatting for an overlapping cell consistent.

**Comment by andmarti1424** [comment]
Yes. Its actually shows the overlapped chars without the actuall cell colors. This would show weird in case you have a background color different on both cells, but it should shown correctly if they both share the same background..
